extensions/amp-story/1.0/media-performance-metrics-service.js

```diff
@@ -218,6 +221,10 @@ export class MediaPerformanceMetricsService {
       TickLabel.VIDEO_CACHE_STATE,
       videoCacheState
     );
+    this.performanceService_.tickDelta(
+      TickLabel.VIDEO_IS_FIRST_PAGE,
+      mediaEntry.metrics.isActivePage ? 1 : 0
+    );
```

Comment:
- Please create an enum for the "is first page" values
- Please use something like `matches(media, `amp-story-page:first-of-type ${media.id}`)` to determine if it's on the first page (and not the active page). No need to worry about the #page in the hash of the URL, this is not used by any platform that reports to the CSI, and is wrong in some specific cases

Reply:
Ok, that's what I was worried about (and why I wanted to route the `isPrerenderedActivePage`). Otherwise, `amp-story-page:first-of-type` works just as well